initCobraToolbox.m

```diff
@@ -171,7 +171,7 @@ function initCobraToolbox()
 %If the Matlab version is prior to Matlab 2016a, we have to use the TC.xml
 %structure to provide autocompletion, otherwise we can use the json
 %definition.
-if MatlabYear < 2015
+if MatlabYear < 2016

```

Comment:
I can't get `tabcomplete` to work on `R2014b` and `R2015b`. I am in favor of supporting `tabcomplete` only for `R2016+` and leaving the condition with `TC.xml` completely out. Do you agree?